test/presenters/trial.js

```diff
@@ -56,6 +56,69 @@ describe('trial presenter', () => {
     });
   });
 
+  describe('records', () => {
+    it('sorts records alphabetically by source_id', () => {
+      const trialAttributes = {
+        records: [
+          {
+            "id": "314912a3-60e5-4056-9ae4-89a37e369ae8",
+            "source_id": "ictrp",
+            "url": "http://localhost:10010/v1/trials/ee6883be-3414-456c-82aa-91d802a83c98/records/314912a3-60e5-4056-9ae4-89a37e369ae8",
+            "source_url": "http://apps.who.int/trialsearch/Trial3.aspx?trialid=EUCTR2010-018661-35-BG",
+            "updated_at": "2016-03-25T06:03:02.677Z"
+          },
+          {
+            "id": "401ef4ea-0dca-4af3-b052-84112b2df8fe",
+            "source_id": "euctr",
+            "url": "http://localhost:10010/v1/trials/ee6883be-3414-456c-82aa-91d802a83c98/records/401ef4ea-0dca-4af3-b052-84112b2df8fe",
+            "source_url": "https://www.clinicaltrialsregister.eu/ctr-search/trial/2010-018662-23/FI",
+            "updated_at": "2016-05-25T22:45:53.280Z"
+          },
+          {
+            "id": "91d1e3eb-e512-47ca-88b9-bb1013e55ef0",
+            "source_id": "nct",
+            "url": "http://localhost:10010/v1/trials/ee6883be-3414-456c-82aa-91d802a83c98/records/91d1e3eb-e512-47ca-88b9-bb1013e55ef0",
+            "source_url": "https://clinicaltrials.gov/ct2/show/NCT01193257",
+            "updated_at": "2016-02-07T03:26:01.242Z"
+          },
+        ]
+      };
+      const trial = trialPresenter(trialAttributes);
+
+      should(trial.records).deepEqual([
```

Comment:
As a suggestion, you could do something like:
```javascript
const recordsSources = trial.records.map((record) => record.source_id);
should(recordsSources).deepEqual(['euctr', 'ictrp', 'nct']);
```

This is because you don't care (on this test) about the other fields, just that the order is correct. This makes the test a bit more resistant to changes in the `records` structure.